db/migrate/20170905095543_add_latest_post_at_to_course_forum_topic.rb

```diff
@@ -0,0 +1,11 @@
+class AddLatestPostAtToCourseForumTopic < ActiveRecord::Migration
+  def change
+    add_column :course_forum_topics, :latest_post_at, :datetime
+
+    Course::Forum::Topic.with_latest_post.each do |topic|
```

Comment:
can you use a SQL query to do this, instead of loading it to ruby, otherwise in production it might be very slow.